R/module_teal.R

```diff
@@ -179,7 +179,7 @@ srv_teal <- function(id, modules, teal_data_rv, filter = teal_slices()) {
         } else if (isTRUE(attr(filter, "module_specific"))) {
           # we should create FilteredData even if modules$datanames is null
           # null controls a display of filter panel but data should be still passed
-          datanames <- if (is.null(modules$datanames) || modules$datanames == "all") {
+          datanames <- if (is.null(modules$datanames) || all(modules$datanames == "all")) {
```

Comment:
```suggestion
          datanames <- if (is.null(modules$datanames) || identical(modules$datanames, "all")) {
```

`all` suggests that there is a possibility to have an input vector with multiple "all" in them. But, the input can be "all", or a vector of datanames other than "all". So, I'd prefer to use `identical` here.

Reply:
I approve this :+1: